app/components/common/SystemCell.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import {
+  View,
+  StyleSheet,
+  TouchableOpacity,
+  ImageBackground,
+  Dimensions,
+} from 'react-native'
+import RNText from './RNText'
+import colors from '../../config/colors'
+
+const windowWidth = Dimensions.get('window').width
+
+const SystemCell = ({ imageUrl, icon, title, onPress }) => {
+  const image = { uri: imageUrl }
+
+  return (
+    <TouchableOpacity
+      onPress={onPress}
+      accessible={true}
+      style={styles.container}
+    >
+      <ImageBackground
+        imageStyle={{ borderRadius: 6 }}
+        source={image}
+        style={styles.image}
+      >
+        <View style={styles.contentContainer}>
+          <View style={styles.icon}>{icon}</View>
```

Comment:
Cool, why pass in a component over passing in a string and it always being an Icon? What happens if there's no icon passed in ie the view will still be rendered with the `paddingRight`?